packages/ui/react-ui-attention/src/testing/decorators/withAttention.ts

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { type Decorator, type StoryFn } from '@storybook/react';
+import { createElement } from 'react';
+
+import { RootAttentionProvider } from '../../components';
+
+export const withAttention: Decorator = (Story: StoryFn) => {
```

Comment:
(hint don't need to type Story since Decorator types the entire function)

Reply:
Better yet, `satisfies Decorator` at the end? Might not give story an inference perhaps though...